cache/cache.go

```diff
@@ -54,7 +48,7 @@ func (c Cache) Restore(src string) error {
 	return nil
 }
 
-func restoreCache(src string, s storage.Storage, a archive.Archive) error {
+func restoreCache(src string, fallback string, s storage.Storage, a archive.Archive) error {
```

Comment:
I think that'll make things easier if we do end up having a more robust fallback strategy. Like if you wanted it to search for the branch this branch was forked from.